lute/static/js/dict-tabs.js

```diff
@@ -0,0 +1,455 @@
+"use strict";
+
+function createDictTabs(num = 0) {
+  // TERM_DICTS.push("*https://glosbe.com/de/en/###");
+  // TERM_DICTS.push("*https://en.langenscheidt.com/german-english/###");
+  // TERM_DICTS.push("*https://en.pons.com/translate/german-english/###");
+  // TERM_DICTS.push("*https://www.collinsdictionary.com/dictionary/german-english/###");
+  // TERM_DICTS.push("https://de.thefreedictionary.com/###");
+  // TERM_DICTS.push("*https://dict.tu-chemnitz.de/deutsch-englisch/###.html");
+  // TERM_DICTS.push("*https://www.translate.ru/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%BD%D0%B5%D0%BC%D0%B5%D1%86%D0%BA%D0%B8%D0%B9-%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9/###");
+
+  if (TERM_DICTS.length <= 0) return;
+
+  let sliceIndex;
+  let columnCount;
+
+  if (num < 0) num = 0;
+
+  if (num == 0 || num == null || num >= TERM_DICTS.length) {
+    sliceIndex = TERM_DICTS.length;
+    columnCount = sliceIndex;
+  } else {
+    sliceIndex = num - 1;
+    columnCount = num;
+  }
+  
+  const TABBED_DICTS = TERM_DICTS.slice(0, sliceIndex);
+  const LISTED_DICTS = TERM_DICTS.slice(sliceIndex);
+
+  const dictTabButtons = new Map();
+  const dictTabsContainer = document.getElementById("dicttabs");
+  const dictTabsLayoutContainer = document.getElementById("dicttabslayout");
+  const iFramesContainer = document.getElementById("dictframes");
+  
+  dictTabsLayoutContainer.style.gridTemplateColumns = `repeat(${columnCount}, minmax(2rem, 8rem))`;
+
+  TABBED_DICTS.forEach((dict, index) => {
+    const domain = getDictURLDomain(dict);
+    const faviconURL = getFavicon(domain);
+    let iFrame = null;
+    let btnLabel = domain.split("www.").splice(-1)[0];
+
+    const btn = createTabBtn(btnLabel, 
+                              dictTabsLayoutContainer, 
+                              index, 
+                              isURLExternal(dict), 
+                              faviconURL);
+
+    if (!isURLExternal(dict)) {
```

Comment:
Do we really need a separate iframe for each dict, given that they're all displayed in the same location?  e.g. if we want to show something else in the frame (per new issue https://github.com/jzohrab/lute-v3/issues/237), will we need to create another iframe?